geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -1837,7 +1836,7 @@ public CommitProcessQueryMessage(Object trackerKey, int processorId) {
 
     @Override
     protected void process(ClusterDistributionManager dm) {
-      final boolean processMsgReceived = txTracker.commitProcessReceived(this.trackerKey, dm);
+      final boolean processMsgReceived = txTracker.commitProcessReceived(this.trackerKey);
```

Comment:
I think the old code was better that passed along the "dm" parameter.

Reply:
txTracker.commitProcessReceived method no longer need distribution manager anymore.